src/index.css

```diff
@@ -1,20 +1,40 @@
 :root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
-  line-height: 1.5;
-  font-weight: 400;
+  /* Colors */
+  --color-myOrange: #FF6200;
+  --color-myOrangeHover: #FF4500;
+  --color-myBlack: #333;
+  --color-myWhite: #fff;
+  --color-mySky: #559bd1;
+  --color-mySky60: #77AFDA;
+  --color-myPrimary: #007bff;
+  --color-myPrimaryHover: #0056b3;
+  --color-mySecondary: #6c757d;
+  --color-mySecondaryHover: #565e64;
+  --color-mySuccess: #28a745;
+  --color-mySuccessHover: #218838;
+  --color-myDanger: #dc3545;
+  --color-myDangerHover: #c82333;
+  --color-myWarning: #ffc107;
+  --color-myWarningHover: #e0a800;
+  --color-myInfo: #17a2b8;
+  --color-myInfoHover: #138496;
+  --color-myLight: #f8f9fa;
+  --color-myLightHover: #dae0e5;
+  --color-myDark: #343a40;
+  --color-myDarkHover: #1d2124;
+  --color-myBrown: #8B4513;
+  --color-myBrownHover: #A0522D;
```

Comment:
Please delete changes

Reply:
The main goal of this file it was prepare some custom colors, i know that Joy has own but I think it will be good place to prepare som custom

Reply:
No way, you can't even use them as const in theme provider, the documentation suggest, to work around only with theme provider. Because especially to prevent using example like below:

```mt: 3,
            background: 'var(--color-myBrown)',
            '&:hover': {
              background: 'var(--color-myBrownHover)',
            },```